src/styles/navbar.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Doppio+One&display=swap');
+
+.navbar{
+    height: 10vh;
+    width: 100%;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-around;
+    font-family: 'Doppio One', 'sans-serif';
+    border-bottom: 1px solid orange;
+}
+
+.logo{
+    height: 80%;
+    width: 10%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 30px;
+    color: orange;
+}
+
+.nav-items{
+    height: 80%;
+    width: 60%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
```

Comment:
Space around nav item components too large
![image](https://user-images.githubusercontent.com/35983270/122278841-47a5a900-cedf-11eb-916a-6d2f562c10e3.png)

Reply:
I've reduced the space around the nav items